### PR TITLE
python3Packages.{segments,clldutils,csvw}: init

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

Part of our (@Mic92 and my) plan to package Mozilla TTS.

Phonemizer (up next) requires segments, which in turn requires clldutils and csvw.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mweinelt** [comment]
Result of `nixpkgs-review pr 98297` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>python37Packages.clldutils</li>
    <li>python37Packages.csvw</li>
    <li>python37Packages.segments</li>
    <li>python38Packages.clldutils</li>
    <li>python38Packages.csvw</li>
    <li>python38Packages.segments</li>
  </ul>
</details>

**Comment by Mic92** [comment]
Result of `nixpkgs-review pr 98297` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages failed to build:</summary>
  <ul>
    <li>python37Packages.clldutils</li>
    <li>python37Packages.csvw</li>
    <li>python37Packages.segments</li>
    <li>python38Packages.clldutils</li>
    <li>python38Packages.csvw</li>
    <li>python38Packages.segments</li>
  </ul>
</details>

**Comment by Mic92** [comment]
One of the checksums seems wrong:

```
hash mismatch in fixed-output derivation '/nix/store/z6sjq2icxw2rf0g2iw34q5qkzyi4qdr9-source':
  wanted: sha256-qeKjcW6SK0c5bmhXuUEHm/62h8UbD0TFiKbBPhVL0Cg=
  got:    sha256-rfnWempSe0V+V7g9tCN640nzEPaYwyPMmzDFtKPOXVU=
```

File: pkgs/development/python-modules/csvw/default.nix [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+buildPythonPackage rec {
+  pname = "csvw";
+  version = "1.8.0";
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner = "cldf";
+    repo = "csvw";
+    rev = "v${version}";
+    sha256 = "0a6h9cakxhd6i32l83qvqn3vdzlv0x0vjmv8dqwlfawjdrqs7qm9";
```
Comment (jonringer) [inline pkgs/development/python-modules/csvw/default.nix:24]:
```suggestion
    sha256 = "0maxrsiv9i9hkg627hwqyq8g6jg3g8iv8gdqaxz4aysjd9xddydd";
```
```
hash mismatch in fixed-output derivation '/nix/store/z6sjq2icxw2rf0g2iw34q5qkzyi4qdr9-source':
  wanted: sha256:0a6h9cakxhd6i32l83qvqn3vdzlv0x0vjmv8dqwlfawjdrqs7qm9
  got:    sha256:0maxrsiv9i9hkg627hwqyq8g6jg3g8iv8gdqaxz4aysjd9xddydd
```

**Comment by mweinelt** [comment]
Hah, odd. Wonder how I messed that up.

**Comment by mweinelt** [comment]
Result of `nixpkgs-review pr 98297` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>python37Packages.clldutils</li>
    <li>python37Packages.csvw</li>
    <li>python37Packages.segments</li>
    <li>python38Packages.clldutils</li>
    <li>python38Packages.csvw</li>
    <li>python38Packages.segments</li>
  </ul>
</details>

**Comment by jonringer** [comment]
> Hah, odd. Wonder how I messed that up.

when I copy from other expressions, I sometimes forget about fixed-point derivation hashes as well

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 98297 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>6 packages failed to build:</summary>  <ul>    <li>python37Packages.clldutils</li>    <li>python37Packages.csvw</li>    <li>python37Packages.segments</li>    <li>python38Packages.clldutils</li>    <li>python38Packages.csvw</li>    <li>python38Packages.segments</li>  </ul></details>

**Pull request merged**